test/live_view_native/swiftui/rules_parser_test.exs

```diff
@@ -259,9 +259,9 @@ defmodule LiveViewNative.SwiftUI.RulesParserTest do
     end
 
     test "to_ime in the middle of a chain" do
-      input = "color(foo.to_ime(color).bar)"
+      input = "color(.foo.to_ime(color).bar)"
```

Comment:
this was correct as-is. Some imes can be chained from somethign else. Technically, you'd see something like `color(color: Color.red)`

Reply:
I am handling the special case of module/class names ([here](https://github.com/liveview-native/liveview-client-swiftui/pull/1149/files#diff-d2bfbf65b14e1f56662259c0c43e3ebd3d0674ccaa279ef368a62c4d4f94b42fR32)). Right now, `color(foo.to_ime(color).bar)` is incorrect but `color(Foo.to_ime(color).bar)` is okay.

Would you want to support both?

Reply:
@NduatiK is the issue in this case that `foo` is lower-case? If so I can check with @carson-katri to see if this format is valid or not

Reply:
@NduatiK let's change this to `Foo.to_ime(color).bar`

Reply:
We can support both, but I expect that we will only have the class name format. It would be strange to see `color(red.to_ime(shade))`. Eventually, we will want to become strict about what is supported so that syntax validation is easier.

Reply:
agreed, you can change this test to the `Foo` format in the PR